caffe2/serialize/versions.h

```diff
@@ -67,7 +67,7 @@ constexpr uint64_t kProducedFileFormatVersion = 0x3L;
 //  0x3L: (Comment missing)
 //  0x4L: (Comment missing)
 //  0x4L: (update) Added schema to function tuple. Forward-compatible change.
-constexpr uint64_t kProducedBytecodeVersion = 0x4L;
+constexpr uint64_t kProducedBytecodeVersion = 0x5L;
```

Comment:
Are we bumping the version in the same PR?

Reply:
Chatted offline. It's for testing purpose. Please remember to change it back to 4 before landing it.

Reply:
Ideally yeah, consider the change is quite minimum and all the test in place.

Reply:
Move the version bump to #57888